shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -504,6 +506,7 @@ public boolean performAction(int virtualViewId, int action, Bundle arguments) {
             case AccessibilityNodeInfo.ACTION_SET_SELECTION: {
                 final Map<String, Integer> selection = new HashMap<>();
                 final boolean hasSelection = arguments != null
+                        && Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN_MR2
```

Comment:
I'd prefer that we mix SDK checks with application logic as little as humanly possible.  Please consider making that check explicit in its own if-statement.

Reply:
This makes sense to me, but unfortunately I've been seeing finicky behavior from the linter where it will report that code branches are executable if the guard clause preventing it is a little too far removed from the line or too abstracted. I'll try reworking these guards to see if I can separate the concerns better and keep the linter happy still, but if there are conflicts I can't avoid I think it's ultimately better to mix the logic than use `@SuppressWarnings` too broadly.